docs/src/openapi-v1.yaml

```diff
@@ -1153,6 +1153,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /transaction/material/{metadataVersion}:
+    get:
+      tags:
+      - transaction
+      summary: Get all the network information needed to construct a transaction offline and
+        the version of metadata specified in `metadataVersion`.
+      description: Returns the material that is universal to constructing any
```

Comment:
"Returns all the materials necessary to constructing any signed transactions offline."

I don't think we need any other details, and we can also get rid of the mention to `/tx/artifacts`.